ethpm_types/ast.py

```diff
@@ -135,7 +135,13 @@ def get_nodes_at_line(self, line_numbers: SourceLocation) -> List["ASTNode"]:
         """
 
         nodes = []
-        if self.line_numbers == line_numbers:
+        if len(line_numbers) != 4:
+            raise ValueError(
+                "Line numbers should given in form "
+                "`(lineno, col_offset, end_lineno, end_coloffset)`"
+            )
+
+        if all([x == y for x, y in zip(self.line_numbers, line_numbers)]):
```

Comment:
Not necessary to wrap in a list
```suggestion
        if all(x == y for x, y in zip(self.line_numbers, line_numbers)):
```